doc/translations/classes.pot

```diff
@@ -28656,8 +28656,8 @@ msgid ""
 "Returns a copy of the given input event which has been offset by "
 "[code]local_ofs[/code] and transformed by [code]xform[/code]. Relevant for "
 "events of type [InputEventMouseButton], [InputEventMouseMotion], "
-"[InputEventScreenTouch], [InputEventScreenDrag], [InputEventMagnifyGesture] "
-"and [InputEventPanGesture]."
+"[InputEventScreenTouch], [InputEventScreenDrag], [InputEventGesturePan], "
+"[InputEventGesturePinth] and [InputEventGestureTwist]."
```

Comment:
It's better to leave the `.pot` and `.po` files unchanged, I synchronize them manually when syncing translations.